python/ray/test/cluster_utils.py

```diff
@@ -15,7 +15,8 @@ class Cluster(object):
     def __init__(self,
                  initialize_head=False,
                  connect=False,
-                 head_node_args=None):
+                 head_node_args=None,
+                 redis_password=None):
```

Comment:
We don't need this, it can just be part of `head_node_args`, right?

Reply:
Not quite. Otherwise, this will fail when passing `connect=True` because `redis_password` isn't passed to `ray.init()`.

I thought this was the best way to proceed, but an alternative is something like
```python
redis_password = head_node_args.get("redis_password")
ray.init(redis_password=redis_password)
```
Do you think that would be better?

Reply:
Oh I think I understand now. Yeah, I think the snippet in your comment is better.